lib/improver/tests/nbhood/nbhood/test_NeighbourhoodProcessing.py

```diff
@@ -123,29 +120,31 @@ def set_up_cube(zero_point_indices=((0, 0, 7, 7),), num_time_points=1,
 
     cube = Cube(data, standard_name="precipitation_amount",
                 units="kg m^-2")
-    coord_system = OSGB()
-    scaled_y_coord = OSGBGRID.coord('projection_y_coordinate')
+
     cube.add_dim_coord(
         DimCoord(
             range(num_realization_points), standard_name='realization'), 0)
     tunit = Unit("hours since 1970-01-01 00:00:00", "gregorian")
     time_points = [402192.5 + _ for _ in range(num_time_points)]
     cube.add_dim_coord(DimCoord(time_points,
                                 standard_name="time", units=tunit), 1)
+
+    y_points = np.linspace(0, 100000, num=51)
```

Comment:
Can't we just create the right y_points based on the number of points requested and the grid spacing